sms-authentication/README.md

```diff
@@ -0,0 +1,53 @@
+# sms-authentication
+
+Authenticate users using Twilio SMS API, Schema Extensions and Graphcool Functions
+
+> Note: Schema Extensions are currently only available in the Beta Program.
+
+## Getting Started
+
+```sh
+npm -g install graphcool
+graphcool init --schema sms-authentication.graphql
+```
+
+## Authentication flow in app
+1. The user enters his/her phone number
+2. Your app receives a phone number
+3. Your app calls the Graphcool mutation `authenticateSmsUser(phone: String!)`
+4. The `authenticateSmsUser` function checks if the user exists, if not, a new `User` will be created
+5. The app will generate an SMS code and a JWT token that will be saved under `User`
+6. Twilio sends that sms code to the phone number provided
+7. Your `User` model is updated with the JWT-encrypted sms code
+8. Your user will receive a text message. Your app will take that sms code, unencrypt it and check to make sure its correct
+
+## Setup the Authentication Function
+
+* Create a new Schema Extension Function and paste the schema from `schema-extension.graphql` and code from `sms-authentication.js`.
```

Comment:
I think setting up the confirm-code SE is missing here, which is why I didn't understand the flow right away...